liberty-maven-plugin/src/main/resources/META-INF/plexus/components.xml

```diff
@@ -41,6 +41,7 @@
                             <prepare-package>
                                 :liberty-maven-plugin:install-server,
                                 io.openliberty.tools:liberty-maven-plugin:create,
+								io.openliberty.tools:liberty-maven-plugin:prepare-feature,
```

Comment:
You should update the doc for the liberty-assembly packaging type to indicate that prepare-feature has been added to the prepare-package phase.

Reply:
Here is the doc I am referencing: https://github.com/OpenLiberty/ci.maven#liberty-assembly